pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -445,16 +436,17 @@ func (b *MasqueradeLibvirtSpecGenerator) generate(domainIface api.Interface) err
 	return nil
 }
 
-func (b *MasqueradeLibvirtSpecGenerator) decorateDomainIfaceSpec(domainIface *api.Interface) error {
+func (b *MasqueradeLibvirtSpecGenerator) domainIfaceSpec() (*api.Interface, error) {
```

Comment:
imo, this method is a **really** good match for having discover in its name, since it reads stuff from netlink.